bin/pycbc_multi_inspiral

```diff
@@ -629,6 +633,8 @@ with ctx:
                         network_out_vals['dec'] = [sky_positions[1][position_index]] * num_events
                         network_out_vals['ra'] = [sky_positions[0][position_index]] * num_events
                         network_out_vals['slide_id'] = [slide] * num_events
+                        network_out_vals['mass1'] = [template.params['mass1']] * num_events
+                        network_out_vals['mass2'] = [template.params['mass2']] * num_events
```

Comment:
The template ID is stored already. Storing the masses is handy for the chirp mass calculation which happens down the line. The old code stored the template ID as well as the masses. I believe it also wrote the spins for each trigger, which I did not find the need for. This might be easier than adding extra dependencies on the template bank file.

Reply:
I don't see the template ID listed in `network_out_vals`.  If we want to avoid the dependency on the template bank, then we should copy all `template.params`.  Otherwise this will be outdated if the template parameters are expanded (e.g., say we have a template bank with tidal deformabilities or precession: then we have to go back and expand this to grab lambda's and spin components).

Reply:
The template ID is written elsewhere (looks like it's done [here](https://github.com/gwastro/pycbc/blob/master/pycbc/events/eventmgr.py#L653)). I've confirmed it shows up in the output files as well.